aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/OnboardApplicationDeepAnalysisCommand.java

```diff
@@ -38,11 +38,16 @@ public class OnboardApplicationDeepAnalysisCommand extends BasicCollable {
             description = "The name of the snapshot to create")
     private String snapshotName;
 
+    @CommandLine.Option(names = {"--sleep-duration"},
+            description = "Amount of time  used to fetch the ongoing job status (specified in seconds ). The default value is: ${DEFAULT-VALUE}",
```

Comment:
I would rephrase this as "Number of seconds ..." to make this message shorter, and avoid the word "amount" that is often used when speaking of a quantity of money.

Reply:
let go for number of seconds